manager/base.py

```diff
@@ -781,9 +781,9 @@ def cyndi_join(query):
     return query
 
 
-def is_not_cacheable_request(args):
+def is_cacheable_request(args):
     """Checks if request needs cyndi join"""
-    return any(
+    return not any(
         key in args and args[key] is not None
```

Comment:
maybe like this to remove more negations? Request is cacheable when all of the listed keys are not present in args
```suggestion
    return all(
       args.get(key) is None
```

Reply:
makes sense, changed